qdev_wrappers/alazar_controllers/acquisition_parameters.py

```diff
@@ -37,10 +37,9 @@ def set_raw(self, value):
             value: value to set the parameter to
         """
         self._check_and_update_instr(value, param_name=self.name)
-        self._save_val(value)
 
     def get_raw(self):
-        return self._latest['value']
+        return self.cache.get()
```

Comment:
instead of this, just `pass get_cmd=None` to the constructor. This way the change will be qcodes backwards compatible.